scripts/helpers.js

```diff
@@ -1,16 +1,39 @@
+var moment = require('moment');
 var urllib = require('url');
 
 var utils = require('../lib/utils');
-
 var pkg = require('../package');
+
+var isUrl = utils.isUrl;
+var urljoin = utils.urljoin;
+
 var aframeCurrentSha = 'master';
 try {
   aframeCurrentSha = pkg.dependencies.aframe.split('#')[1];
-} catch (e) {
-}
+} catch (e) {}
 
-var isUrl = utils.isUrl;
-var urljoin = utils.urljoin;
+hexo.extend.helper.register('blog_attribution', function (author) {
+  // Twitter handle.
+  if (author[0] === '@') {
+    return '<a class="blog-attribution" href="https://twitter.com/' + author + '">' +
+      author + '</a>';
+  }
+  // Make a space imply that it is not a Github username. Return as is.
+  if (author.indexOf(' ') !== -1) {
```

Comment:
1. Create a file called `src/_data/known_github_users.json` containing this:

```json
[
  "carpedmt",
  "chrismwaite",
  "dkraeker",
  "donmccurdy",
  "drryanjames",
  "hellocreep",
  "i3games",
  "joshgalvin",
  "Lullabot",
  "meta-meta",
  "ngokevin",
  "withersc",
  "ybinstock"
]
```
2. Change this single line to
    ```js
    if (site.known_github_users.indexOf(author) !== -) {
    ```

3. Woo!

<hr>

I'm not going to suggest this, but you could automate this by creating a function (perhaps as a script `bin/updateGitHubUsers.js` or something in this file) that looks the author properties of each project item in `twia.projects` - and then uses [`gh-user`](https://www.npmjs.com/package/gh-user) to look up whether the user exists on GitHub, and writes out known users to `src/_data/github_users.json`.